### PR TITLE
Added route to fetch openapi.json

### PR DESCRIPTION
**Pull request by hsong-rh**

### CODE REVIEW

File: app/controllers/api/v0x1/root_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Api
+  module V0x1
+    class RootController < ApplicationController
+      def openapi
+        render :json => Rails.root.join('public', 'approval', 'v0.1.0', 'openapi.json').read
```
Comment (gmcculloug) [inline app/controllers/api/v0x1/root_controller.rb]: We should wait until https://github.com/ManageIQ/approval-api/pull/73 is merged and update this to `v1.0.0`.

Additionally, based on https://github.com/ManageIQ/catalog-api/pull/177#discussion_r270151187 should we drop the third digit and just keep major/minor?  cc @syncrou

**Comment by bzwei** [comment]
Closing this PR in favor of #73

**Comment by gmcculloug** [comment]
@bzwei This PR was adding functionality that is not in PR #73.  Should it be closed or was it meant to be updated after that PR is merged.  (btw - I prefer we keep the two as separate PRs and not combine them if this is still requried.)

**Comment by bzwei** [comment]
Sorry. I accidentally closed. Reopening.

**Comment by gmcculloug** [comment]
@hsong-rh Please rebase and adjust paths.

**Pull request merged**